Scripts/AppCenter/build-appcenter-ios-framework.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# Load local .bash_profile to have access to MS_ARM64E_XCODE_PATH
+source ~/.bash_profile
```

Comment:
Should we do this in the script? It is obvious that any of changes made in `profile`s wouldn't be applied right away.
When customers add environment variable to their `profile` files, they should run `source` to apply the changes.
It isn't a big deal to us but users might have some other changes in their `profile` files which are unrelated to App Center.

Reply:
It doesn't work if you already have Xcode open and put an environment variable in Terminal after that. Once you reopen Xcode, it should be able to read the variable. This is just one time thing so we don't need to execute `source` to reload `.bash_profile`.
Just imagine when you install Android Studio without `JAVA_HOME`. You should close your IDE and open it again after setting the variable. I can explain this offline.

Reply:
I also synced with @clpolet. Then my suggestion would be using another file other than `.bash_profile`.
You can add a new file in the root of the repo and add the file to `.gitignore`. Then export MS_ARM64E_XCODE_PATH in the new file as you did in `~/.bash_profile` and `source the_new_file` in the script.